src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -59,6 +61,33 @@ def dcos_browse(name, resource_group_name):
 
     return
 
+def dcos_install():
```

Comment:
`install_location` should be an optional argument to this function so it can be specified on the command line with `acs dcos install --install-location ...`

i.e.
```
def dcos_install(install_location=None):
   ...
```